### PR TITLE
Form designer wysiwyg translations

### LINKED ISSUE #2475: Form designer FormIO translations UI: WYSIWYG content field translations
Related issue: https://github.com/open-formulieren/open-forms/issues/2315

Issue raised in https://github.com/open-formulieren/open-forms/pull/2437

The `vrije tekst` field is a WYSIWYG content field, which will not work properly with the default translations setup implemented in the above PR. It should probably have a different translations tab that only shows one WYSIWYG editor per language, because only the content is translatable.

Question: will this content be picked up by FormIO's translation engine, or should we translate it when serializing the FormStepDefinition?

### PR DESCRIPTION
**Pull request by CharString**
Closes #2475

### CODE REVIEW

File: src/openforms/js/utils/translation.js [hunk 1]
```diff
@@ -1,4 +1,8 @@
+import FormioUtils from 'formiojs/utils';
```
Comment (CharString) [inline src/openforms/js/utils/translation.js:1]: The form
```javascript
import {Utils as FormioUtils} from 'formiojs';
```

trips jest with `SyntaxError: Unexpected token 'export'`. This phrasing doesn't. [Other workarounds exist](https://www.sobyte.net/post/2022-06/jest/)

File: src/openforms/js/components/form/content.js [hunk 2]
```diff
@@ -26,19 +26,80 @@ const CONTENT_PRESENTATION = {
...
+                  type: 'tabs',
+                  components: LANGUAGES.map(([languageCode, _label]) => {
+                    return {
+                      key: languageCode,
+                      label: languageCode.toUpperCase(),
+                      components: [
+                        {
+                          label: 'Literal',
+                          hideLabel: true,
+                          key: `of-translations.${languageCode}[0].literal`,
```
Comment (CharString) [inline src/openforms/js/components/form/content.js:73]: This magnificent key makes it such that it looks like the datagrid used else where and gets picked up into the componentTranslations on PUT and POST.

File: src/openforms/js/components/form/content.js [hunk 3]
```diff
@@ -26,19 +26,80 @@ const CONTENT_PRESENTATION = {
...
+                          input: true,
+                          unique: true,
+                          type: 'hidden',
+                          customDefaultValue: ({instance, data}) =>
+                            instance.options.editComponent.html,
+                        },
+                        {
+                          label: 'Translation',
+                          hideLabel: true,
+                          key: `of-translations.${languageCode}[0].translation`,
```
Comment (CharString) [inline src/openforms/js/components/form/content.js:83]: The beauty grows on you. :wink:

File: src/openforms/js/components/form/content.js [hunk 4]
```diff
@@ -26,19 +26,80 @@ const CONTENT_PRESENTATION = {
...
-      rows: 3,
-      tooltip: 'The HTML template for the result data items.',
+      type: 'columns',
+      key: 'content',
+      columns: [
+        {
+          components: [
+            {
+              type: 'tabs',
+              components: [{label: 'Content', components: [CONTENT_EDITOR]}],
```
Comment (CharString) [inline src/openforms/js/components/form/content.js:52]: Single tab. The easiest way to make it align and look similar side by side to the translations.

**Review by stevenbal — APPROVED** [review]

**Pull request merged**